tests/testthat/test-nm-tables.R

```diff
@@ -0,0 +1,61 @@
+
+withr::local_options(list("bbr.verbose" = FALSE))
+
+test_that("nm_tables() works [BBR-NMT-001]", {
+  res <- nm_tables(MOD1)
+  expect_equal(length(res), length(MOD1_TABLE_FILES) + 1)
+  expect_equal(res$data, nm_data(MOD1))
+  expect_equal(res$tab, nm_tab(MOD1))
+  expect_equal(res$par.tab, nm_par_tab(MOD1))
+})
+
+test_that("nm_tables() works with .files argument [BBR-NMT-001]", {
+  res <- nm_tables(MOD1, .files = "1.tab")
+  expect_equal(length(res), 2)
+  expect_equal(res$data, nm_data(MOD1))
+  expect_equal(res$tab, nm_tab(MOD1))
+})
+
+test_that("nm_tables() naming works correctly [BBR-NMT-002]", {
+  # construct fake tables with test case names
+  table_lines <- c(
+    "TABLE NO. 1",
+    "NUM VAR",
+    "1 2"
+  )
+  table_names <- file.path(
+    get_output_dir(MOD1),
+    c("satab1", "catab.1", "par-tab")
```

Comment:
> "satab1", "catab.1"

Are these motivated by actual naming conventions/practice?  Glancing bbr-nonmem-poppk-foce, I didn't spot anything following this pattern.

Reply:
Yes. Example in [here](https://uupharmacometrics.github.io/xpose4/reference/read.nm.tables.html) where the show the roots, but @callistosp said they often also append the "run number" (`model_id` for us) on the _end_ instead of the beginning.

Reply:
(to your point, not a convention we use at Metrum but Sam says it's pretty wide-spread.)